### PR TITLE
Various tree fixes

### PR DESCRIPTION
**Pull request by FinalAngel**
- Adds root copy/cut capabilities
- Hide paste element on cutting element
- Disable CMD+move which triggered a copy mechanism
- Disable selection of multiple entries
- Fix copy and paste permission prompt background color
- Add copy and paste dimmer when active
- Fix helpers display when expanding / collapsing tree nodes
- Fixed menu icon logic (they were reversed)
- Fixed issue with copy and paste not working (no obj.target support)

Root paste position:
image

Copy and paste permission check:
image

### CODE REVIEW

File: cms/static/cms/js/modules/cms.pagetree.js [hunk 1]
```diff
@@ -329,6 +323,61 @@ var CMS = window.CMS || {};
         },
 
         /**
+         * Helper to process the cut and copy events.
+         *
+         * @method _cutAndCopy
+         * @param {Object} [opts]
+         * @param {Number} [opts.type] either 'cut' or 'copy'
```
Comment (vxsx) [inline cms/static/cms/js/modules/cms.pagetree.js]: so the method name should be `cutOrCopy`?

File: cms/static/cms/js/modules/cms.pagetree.js [hunk 2]
```diff
@@ -329,6 +323,61 @@ var CMS = window.CMS || {};
...
+         * @return {Object} event originated event handler
+         */
+        _paste: function _paste(event) {
+            // hide helpers after we picked one
+            this._hideHelpers();
+
+            var copyFromId = this._getNodeId(this.cacheTarget);
+            var copyToId = this._getNodeId($(event.currentTarget));
+
+            // copyToId contains `jstre-1`, assign to root
```
Comment (vxsx) [inline cms/static/cms/js/modules/cms.pagetree.js]: typo jstre-1

File: cms/static/cms/js/modules/cms.pagetree.js [hunk 3]
```diff
@@ -480,9 +534,9 @@ var CMS = window.CMS || {};
          * @return {String} jsTree node element id
```
Comment (vxsx) [inline cms/static/cms/js/modules/cms.pagetree.js]: `{String|Boolean}` now

**Comment by vxsx** [comment]
overall LGTM

**Comment by czpython** [comment]
Backend Ok to merge.

**Pull request merged**